src/session/sessionStore.ts

```diff
@@ -46,27 +47,36 @@ export const inferSessionQuery = (
   if (!req.url) {
     return
   }
-  const url = new URL(req.url)
-  const spaceId = url.searchParams.get('space_id')
-  const userId = url.searchParams.get('user_id')
-  if (spaceId && userId) {
-    return {
-      spaceId,
-      userId,
-    }
+
+  const sessionFromUrl = getSessionFromUrl(req.url)
+
+  if (sessionFromUrl) {
+    return sessionFromUrl
   }
 
   if (req.headers.referer) {
-    const refererUrl = new URL(req.url)
```

Comment:
Here there was probably a typo instead of passing `req.url` --> `req.headers.referer`

Reply:
Thank you